pkgs/README.md

```diff
@@ -111,7 +111,7 @@ Now that this is out of the way. To add a package to Nixpkgs:
 
    Some notes:
 
-   - Add yourself as the maintainer of the package.
+   - Add yourself as the maintainer of the package, in a separate commit.
```

Comment:
In the same commit actually. But adding yourself to `maintainer-list.nix` should happen in a separate commit.

Reply:
ooop yeah thats what i wanted to say